src/index.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+// ******* wave 2 *******
+const defaultCity = 'Seattle';
+let number = parseInt(document.querySelector('.number').innerHTML);
```

Comment:
These should be moved to a callback that fires once the DOM content is loaded. Otherwise there's a risk that the elements may not have been created yet.